addon/services/syncer.js

```diff
@@ -313,6 +316,29 @@ export default Ember.Service.extend({
     });
   },
 
+  /**
+   * Return name of projection to sync up by model name.
+   * @method getSyncUpProjectionName
+   * @param {String} modelName Name of model to get sync up projection name.
+   * @returns Name of projection to sync up.
+   */
+  getSyncUpProjectionName(store, modelName) {  
```

Comment:
Метод `getSyncUpProjectionName` действительно нужен в качестве публичного? Мне бы хватило приватного.
```suggestion
  getSyncUpProjectionName(store, modelName) {
```

Reply:
Думаю, что да. Это позволит переопределить его на прикладном проекте, если потребуется, а потребоваться может